docs/rules/no-mixed-operators.md

```diff
@@ -0,0 +1,125 @@
+# Disallow mixes of different operators (no-mixed-operators)
+
+Enclosing complex expressions by parentheses makes developer's intention clarifying, then the code would get more readability.
+This rule warns that there are different operators consecutively without parentheses *in AST*.
```

Comment:
This rule warns when different operators are used consecutively without parentheses in an expression.